guides/common/modules/proc_increasing-host-limit-for-pull-based-rex-transport.adoc

```diff
@@ -0,0 +1,34 @@
+[id="Increasing_host_limit_for_pull-based_REX_transport_{context}"]
+= Increasing Host Limit for Pull-Based REX Transport
+
+{Project} has a pull-based transport that has the features of REX but with a deployment model similar to katello-agent. 
+This transport uses MQTT as its messaging protocol and includes an MQTT client running on each host.
+To tune the MQTT server `mosquitto` and increase the number of hosts connected to it, follow this procedure on your {ProjectServer}:
```

Comment:
```suggestion
Use this procedure to tune the MQTT server `mosquitto` and increase the number of hosts connected to it:
```

Reworded because the first step affects both Foreman Server and Smart Proxy Server.